src/colvaratoms.h

```diff
@@ -103,6 +98,12 @@ class colvarmodule::atom {
     vel = grad = total_force = cvm::rvector(0.0);
   }
 
+  /// Index of this atom in the internal arrays
+  inline int array_index() const
```

Comment:
It looks like that I need to synchronize the changes to https://github.com/Colvars/colvars/pull/788.